openedx/adg/lms/branding_extension/helpers.py

```diff
@@ -0,0 +1,45 @@
+"""
+Helpers for branding_extension app
+"""
+from datetime import datetime
```

Comment:
Just want to confirm as I am seeing inconsistency everywhere(Not specifically in your PR) and not able to find what to do. Should we add a blank line between module docstring and imports?